pkg/scheduler/scheduler.go

```diff
@@ -126,6 +126,7 @@ func (s *Scheduler) schedule(ctx context.Context) {
 			continue
 		}
 		s.requeueAndUpdate(log, ctx, &w, "workload didn't fit")
+		s.recorder.Eventf(w.Obj, corev1.EventTypeNormal, "Pending", "workload didn't fit")
```

Comment:
Tips: It is better to  define `"workload didn't fit"` as a string to avoid redundancy.